File: C++/숫자의 개수.cpp
#include <iostream>

int main() {
    int num[10] = {0, };
    int a, b, c, i;
    
    std::cin >> a >> b >> c;
    i = a*b*c;
    
    while(i>0){
        num[i%10]++;
        i /= 10;
    }
    
    for(i = 0; i<10;i++){
        std::cout << num[i] << "\n";
    }
    return 0;
}

File: Java/숫자의 개수.java
import java.util.*;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int [] num = new int [10];

        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        int i = a*b*c;
        
        while(i >0){
            num[i%10]++;
            i /= 10;
        }

        for(i = 0; i < num.length; i++){
            System.out.println(num[i]);
        }
    }
}
